src/main/resources/db/changelog/changes/v3.0/add-authority-source-protocol.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+
+  <changeSet id="MODELINKS-188@@add-base-url-protocol-column-to-authority-source-file-table" author="psmagin">
+    <preConditions onFail="MARK_RAN">
+      <and>
+        <tableExists tableName="authority_source_file"/>
+        <not>
+          <columnExists tableName="authority_source_file" columnName="base_url_protocol"/>
+        </not>
+      </and>
+    </preConditions>
+
+    <comment>Add base_url_protocol column to authority_source_file table</comment>
+
+    <addColumn tableName="authority_source_file">
+      <column name="base_url_protocol" type="VARCHAR(10)"/>
```

Comment:
To be safe :)
Do you suggest to reduce it to 5?

Reply:
Yes, I guess 5 should be enough because those are protocols for url so I don't think there will be any other protocols. Also "://" is added on url construction in code so no need to store it either.
So I guess safe to save some space